### PR TITLE
vendor: update golang.org/x/sys 52ab431487773bc9dd1b0766228b1cf3944126bf

### PR DESCRIPTION
**Pull request by thaJeztah**
- ~vendor: update opencontainers/selinux v1.3.1~
  ~full diff: https://github.com/opencontainers/selinux/compare/5215b1806f52b1fcc2070a8826c542c9d33cd3cf...v1.3.1~
- vendor: update golang.org/x/sys 52ab431487773bc9dd1b0766228b1cf3944126bf
  full diff: https://github.com/golang/sys/compare/c990c680b611ac1aeb7d8f2af94a825f98d69720...52ab431487773bc9dd1b0766228b1cf3944126bf

### CODE REVIEW

File: vendor.conf [hunk 1]
```diff
@@ -10,7 +10,7 @@ go.etcd.io/bbolt a0458a2b35708eef59eb5f620ceb3cd1c01a824d # v1.3.3
 google.golang.org/grpc 39e8a7b072a67ca2a75f57fa2e0d50995f5b22f6 # v1.23.1
 google.golang.org/genproto d80a6e20e776b0b17a324d0ba1ab50a39c8e8944
 golang.org/x/text 19e51611da83d6be54ddafce4a4af510cb3e9ea4
-golang.org/x/sys c990c680b611ac1aeb7d8f2af94a825f98d69720 https://github.com/golang/sys
+golang.org/x/sys 52ab431487773bc9dd1b0766228b1cf3944126bf https://github.com/golang/sys
```
Comment (thaJeztah) [inline vendor.conf]: This matches the updated version in opencontainers/selinux; https://github.com/opencontainers/selinux/blob/v1.3.1/go.mod#L7

**Comment by thaJeztah** [comment]
Yeah, I asked them to tag the version that was used in the latest runc release (as it had a security fix), but they tagged the commit after that 🤷‍♂

**Comment by AkihiroSuda** [comment]
needs rebase

**Comment by dims** [comment]
@thaJeztah we updated to 1.3.3 of selinux in https://github.com/containerd/cri/pull/1407/files

**Comment by thaJeztah** [comment]
Rebased: dropped the first commit

**Pull request merged**